packages/hoprd/src/index.ts

```diff
@@ -279,59 +280,70 @@ async function main() {
   // 2. Create node instance
   try {
     logs.log('Creating HOPR Node')
-    node = createHoprNode(peerId, options)
+    node = createHoprNode(peerId, options, false)
     logs.logStatus('PENDING')
     node.on('hopr:message', logMessageToNode)
+    node.on('hopr:connector:created', () => {
+      // 2.b - Connector has been created, and we can now trigger the next set of steps.
+      logs.log('Connector has been loaded properly.')
+      node.emit('hopr:monitoring:start')
+    })
+    node.on('hopr:monitoring:start', async () => {
+      // 3. start all monitoring services, and continue with the rest of the setup.
```

Comment:
Why are APIs called monitoring services?

Reply:
Good question, sadly that was used from past references. We can probably find a better name.